benchmarks/tracking_performances_dis/analysis/trk_dis_plots.cxx

```diff
@@ -0,0 +1,254 @@
+    auto frame_e1 = c1e->DrawFrame(-4,0,4,1.2*h1a1->GetMaximum());
+    frame_e1->GetXaxis()->SetTitle("#eta_{gen} or #eta_{rec}");frame_e1->GetXaxis()->CenterTitle();
+    h1a2->Draw("same");
+    h1b2->Draw("P same");
+
+    TLegend *leg1e = new TLegend(0.15,0.675,0.6,0.875);
+    leg1e->SetBorderSize(0);leg1e->SetFillStyle(0);
+    leg1e->SetHeader(Form("Pythia8: %dx%d GeV, Q^{2} > %d GeV^{2}",ebeam,pbeam,Q2_min));
+    leg1e->AddEntry("h1a2","Generated charged particles w/ P_{T} > 500 MeV/c","fl");
+    leg1e->AddEntry("h1b2","Real-seeded tracks w/ P_{T} > 500 MeV/c","p");
+    leg1e->Draw();
+
+    //Comparison 2b
+    TCanvas *c1e1 = new TCanvas("c1e1");
+    frame_e1->Draw();
+    h1a2->Draw("same");
+    h1c2->Draw("P same");
+
+    TLegend *leg1e1 = new TLegend(0.15,0.675,0.6,0.875);
+    leg1e1->SetBorderSize(0);leg1e1->SetFillStyle(0);
+    leg1e1->SetHeader(Form("Pythia8: %dx%d GeV, Q^{2} > %d GeV^{2}",ebeam,pbeam,Q2_min));
+    leg1e1->AddEntry("h1a2","Generated charged particles w/ P_{T} > 500 MeV/c","fl");
+    leg1e1->AddEntry("h1c2","Truth-seeded tracks w/ P_{T} > 500 MeV/c","p");
+    leg1e1->Draw();
+
+    //Comparison 1 ratio
+    TCanvas *c1f = new TCanvas("c1f");
+    h1rb1->Draw();
+    h1rc1->Draw("same");
+
+    TLegend *leg1f = new TLegend(0.575,0.25,0.875,0.45);
+    leg1f->SetBorderSize(0);leg1f->SetFillStyle(0);
+    leg1f->SetHeader(Form("Pythia8: %dx%d GeV, Q^{2} > %d GeV^{2}",ebeam,pbeam,Q2_min));
+    leg1f->AddEntry("h1rb1","Real-seeded tracking","l");
+    leg1f->AddEntry("h1rc1","Truth-seeded tracking","l");
+    leg1f->Draw();
+
+    //Comparison 2 ratio
+    TCanvas *c1g = new TCanvas("c1g");
+    h1rb2->Draw();
+    h1rc2->Draw("same");
+
+    TLegend *leg1g = new TLegend(0.575,0.25,0.875,0.45);
+    leg1g->SetBorderSize(0);leg1g->SetFillStyle(0);
+    leg1g->SetHeader(Form("Pythia8: %dx%d GeV, Q^{2} > %d GeV^{2}",ebeam,pbeam,Q2_min));
+    leg1g->AddEntry("h1rb2","Real-seeded tracking","l");
+    leg1g->AddEntry("h1rc2","Truth-seeded tracking","l");
+    leg1g->Draw();
+
+    //Hit-based associations -- real-seeded tracks
+    TCanvas *c2a = new TCanvas("c2a");
+    c2a->SetLogy();
+    h2a->Draw();
+
+    TLegend *leg2a = new TLegend(0.25,0.6,0.6,0.875);
+    leg2a->SetBorderSize(0);leg2a->SetFillStyle(0);
+    leg2a->SetHeader(Form("Pythia8: %dx%d GeV, Q^{2} > %d GeV^{2}",ebeam,pbeam,Q2_min));
+    leg2a->Draw();
+
+    //Hit-based associations -- truth-seeded tracks
+    TCanvas *c2b = new TCanvas("c2b");
+    c2b->SetLogy();
+    h2b->Draw();
+
+    TLegend *leg2b = new TLegend(0.25,0.6,0.6,0.875);
+    leg2b->SetBorderSize(0);leg2a->SetFillStyle(0);
+    leg2b->SetHeader(Form("Pythia8: %dx%d GeV, Q^{2} > %d GeV^{2}",ebeam,pbeam,Q2_min));
+    leg2b->Draw();
+
+    //--------------------------------------------------------------------------------------------------------------------------------------------
+
+    // Print plots to pdf file
+    c1a->Print(fmt::format("{}.pdf", output_prefix).c_str());

```

Comment:
Hi @veprbl. I think the opening and closing brackets are needed to save mulitple pages to the same pdf file. Without those, it just replaces each page. That's what seems to happen here: https://eicweb.phy.anl.gov/EIC/benchmarks/detector_benchmarks/-/jobs/3831765/artifacts/browse/results/tracking_performances_dis/epic_craterlake_tracking_only/pythia8NCDIS_18x275_minQ2=1_combined_5/

Reply:
Okay. Thanks for explaining.